### PR TITLE
Update Sample App to `CommunityToolkit.Maui.Markup` v1.0.0-pre9

### PR DESCRIPTION
**Pull request by brminnick**
### Description of Change ###

This PR updates the Sample App to use `CommunityToolkit.Maui.Markup` v1.0.0-pre9.
 
 ### Additional information ###

It also adds `WeakEventManager` to `MauiPopup.PopoverDelegate.PopoverDismissedEvent` now that `WeakEventManager` lives in the `Microsoft.Maui` namespace.

(`WeakEventManager ` was previously in the `Microsoft.Maui.Controls` namespace).

### CODE REVIEW

File: samples/CommunityToolkit.Maui.Sample/Pages/Base/BaseGalleryPage.cs [hunk 1]
```diff
@@ -40,12 +40,10 @@ async void HandleSelectionChanged(object? sender, SelectionChangedEventArgs e)
 		var collectionView = (CollectionView)sender;
 		collectionView.SelectedItem = null;
 
-		if (e.CurrentSelection.FirstOrDefault() is not SectionModel sectionModel)
+		if (e.CurrentSelection.FirstOrDefault() is SectionModel sectionModel)
 		{
-			return;
+			await Shell.Current.GoToAsync(AppShell.GetPageRoute(sectionModel.ViewModelType));
```
Comment (pictos) [inline samples/CommunityToolkit.Maui.Sample/Pages/Base/BaseGalleryPage.cs:45]: If I'm not wrong it's safe to add the ConfigureAwait here
```suggestion
			await Shell.Current.GoToAsync(AppShell.GetPageRoute(sectionModel.ViewModelType)).ConfigureAwait(false);
```

**Review by pictos — APPROVED** [review]
I've one suggestion but it doesn't invalidate this PR.

I disabled the Auto-merge to give a chance to think about it

File: samples/CommunityToolkit.Maui.Sample/Pages/Base/BaseGalleryPage.cs (hunk 1 above)
Comment (brminnick) [reply samples/CommunityToolkit.Maui.Sample/Pages/Base/BaseGalleryPage.cs:45]: You're not wrong; adding `.ConfigureAwait(false)` here would add a performance improvement. 

As a personal rule-of-thumb when implementing an MVVM architecture, I try to avoid using `.ConfigureAwait(false)` in the View layer since the View code typically _does_ require the UI Thread. 

Since `.ConfigureAwait(false)` does not return to the UI thread, any code added afterwards in a future PR would not be on the main thread. 

In the Services + ViewModel layers, however, I _always_ use `.ConfigureAwait(false)` 💯 

Long story short, I try to avoid `.ConfigureAwait(false)` in the View layer to avoid future bugs where future code may need to be running on the UI Thread.

Comment (pictos) [reply samples/CommunityToolkit.Maui.Sample/Pages/Base/BaseGalleryPage.cs:45]: Got it, yeah I suggested it because this is the last async call in the method, so doesn't matter which thread will run. But I understand your point, in terms that this might lead developers - who use this example as a reference - to dragons
